heroic-core/src/main/java/com/spotify/heroic/http/status/StatusResource.java

```diff
@@ -65,15 +73,21 @@ public StatusResource(
     public Response get() {
         final StatusResponse.Consumer consumers = buildConsumerStatus();
         final StatusResponse.Backend backends = buildBackendStatus();
-        final StatusResponse.MetadataBackend metadataBackends = buildMetadataBackendStatus();
+        final StatusResponse.Backend metadataBackends = buildMetadataBackendStatus();
+        final StatusResponse.Backend suggestBackends = buildSuggestBackendStatus();
 
         final StatusResponse.Cluster cluster = buildClusterStatus();
 
         final boolean allOk =
-            consumers.isOk() && backends.isOk() && metadataBackends.isOk() && cluster.isOk();
+            consumers.isOk()
+            && backends.isOk()
+            && metadataBackends.isOk()
+            && suggestBackends.isOk()
```

Comment:
would removing this check help out with the restarts? or you're just separating stuff out first to test?

Reply:
would adding suggest cause the status endpoint to fail if nots no longer `ok`? this end point is used in external health checks

Reply:
Ooh good point. I'll update this, maybe make it configurable what causes a failure.

Reply:
I dug into this more and this doesn't actually check the health of elasticsearch. For suggest, the `isOk` is passed through to the `isReady` method of the `Managed<Connection>` object. The `Managed` container creates and tears down the class in an async thread, only calling the constructor for the `Connection` class.

Reply:
but if ES suggest was red wouldn't the `Connection` not become resolvable? I guess this is the behavior you couldn't reproduce in https://github.com/spotify/heroic/issues/554?